components/navigation/navigation.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="navigation.css">
+    <title>Navigation</title>
+</head>
+
+<body>
+
+    <!-- Header Navigation -->
+    <header class="header">
+        <div class="header__wrapper p-sm-vr">
+            <div class="header__title p-sm-hr">
+                <h3><a href="#">Brand Name</a></h3>
+            </div>
+            <nav class="header__action">
+                <button class="btn btn--primary-link p-xs-vr p-sm-hr rounded-sm semibold-font m-sm-r">
+                    <a class="nav-link nav-link--active" href="#">Link1</a>
+                </button>
+                <button class="btn btn--primary-link p-xs-vr p-sm-hr rounded-sm semibold-font m-sm-r">
+                    <a class="nav-link" href="#">Link2</a>
+                </button>
+                <button class="btn btn--primary-link p-xs-vr p-sm-hr rounded-sm semibold-font m-sm-r">
+                    <a class="nav-link" href="#">Link3</a>
+                </button>
+            </nav>
+        </div>
+        <img src="/assets/images/Ayushman.jpg" class="avatar avatar--sm rounded-circle m-md-r" alt="Avatar" />
+    </header>
+
+    <div class="menu">
+        <button class="menu__button center-content btn btn--primary-text p-xs-vr p-sm-hr rounded-circle semibold-font">
+            <i class="material-icons" aria-hidden="true">
+                menu
+            </i>
+        </button>
+    </div>
+
+    <!-- Siderbar navigation -->
+    <div class="content">
+        <aside class="drawer drawer--permanent">
+            <nav>
+                <ul class="list">
+                    <div class="m-xs-all">
+                        <h4>Heading-1</h4>
+                    </div>
+                    <li class="list__item list__item--active p-xs-vr p-sm-hr m-xs-all">
+                        <a class="list__link" href="#">Link-1</a>
+                    </li>
+                    <li class="list__item p-xs-vr p-sm-hr m-xs-all">
+                        <a class="list__link" href="#">Link-2</a>
+                    </li>
+                    <div class="m-xs-all ">
+                        <h4>Heading-2</h4>
+                    </div>
+                    <li class="list__item p-xs-vr p-sm-hr m-xs-all">
+                        <a class="list__link" href="#">Link-3</a>
+                    </li>
+                    <li class="list__item  p-xs-vr p-sm-hr m-xs-all">
+                        <a class="list__link" href="#">Link-4</a>
+                    </li>
+                </ul>
+            </nav>
+        </aside>
+        <main>
+        </main>
```

Comment:
You could avoid empty tags unless they have specific use. If you want to use it in the future, then you might add it later.

Reply:
Thanks for suggestion will correct it!